src/main/java/at/hannibal2/skyhanni/features/garden/contest/JacobContestStatsSummary.kt

```diff
@@ -25,6 +43,78 @@ class JacobContestStatsSummary {
         }
     }
 
+    @SubscribeEvent
+    fun onTabListUpdate(event: TabListUpdateEvent) {
+//         println("enabled: ${isEnabled()} incontest: ${FarmingContestAPI.inContest}")
+        if (!isEnabled()) return
+        if (!FarmingContestAPI.inContest) return
+//         println("tab list updated")
+        for (line in event.tabList) {
+            tabContestPattern.matchMatcher(line) {
+//                 println(group("crop"))
+                if (group("crop") == FarmingContestAPI.contestCrop?.cropName) {
+                    percent = group("percent").toDouble() / 100
+                    participants = group("participants").formatNumber().toInt()
+
+//                     println("percent: $percent participants: $participants")
+                }
+            }
+        }
+    }
+
+    @SubscribeEvent
+    fun onScoreBoardUpdate(event: ScoreboardChangeEvent) {
+        if (!isEnabled()) return
+
+        for (line in event.newList) {
+            println(line)
+            scoreboardContestTimeLeftPattern.matchMatcher(line) {
+                println(group("crop"))
+                println(group("time"))
+            }
+            scoreboardContestScorePattern.matchMatcher("line") {
```

Comment:
ok just noticed i "line" is as text here, but that isn't the problem.
I was testing stuff and messed this up